### ISSUE TITLE
Release with open version of SuperPoint?

### ISSUE THREAD

**Issue by hihosilvers** [cvg/LightGlue#96]
Hi, wondering if it'd be possible to add the open version of SuperPoint (https://github.com/rpautrat/SuperPoint) to this LightGlue implementation like what you guys have in Glue Factory? 

I'd like to train LightGlue myself on this version of SuperPoint but don't have the resources :(

**Comment by Phil26AT** [comment]
Hi @hihosilvers, sorry for the late reply. This is still on our (quite long) ToDo list (see https://github.com/cvg/glue-factory/issues/22) but needs more tuning. We are welcoming any help here.

**Comment by hihosilvers** [comment]
Thanks. I'll close this issue then

**Comment by noahzn** [comment]
@Phil26AT Hi, do you have any suggestions on training the open SP+LG? Can I directly use the .yaml file in the repo?

**Comment by Phil26AT** [comment]
Hi @noahzn

Yeah I'd recommend starting there. We obtained good results on homography pretraining with it, but it did not translate 1:1 to MegaDepth for us. You could try tuning the learning rate (which we never did with SP-open), or add some data augmentations for better generalization. If you manage to get good results, we'd welcome a contribution here. :)
